static/app/components/group/groupSummary.tsx

```diff
@@ -0,0 +1,135 @@
+import styled from '@emotion/styled';
+
+import FeatureBadge from 'sentry/components/badge/featureBadge';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import Panel from 'sentry/components/panels/panel';
+import * as SidebarSection from 'sentry/components/sidebarSection';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import marked from 'sentry/utils/marked';
+import {type ApiQueryKey, useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface GroupSummaryProps {
+  groupId: string;
+}
+
+interface GroupSummaryData {
+  groupId: string;
+  impact: string;
+  summary: string;
+}
+
+const makeGroupSummaryQueryKey = (
+  organizationSlug: string,
+  groupId: string
+): ApiQueryKey => [`/organizations/${organizationSlug}/issues/${groupId}/summarize/`];
+
+export function GroupSummary({groupId}: GroupSummaryProps) {
+  const organization = useOrganization();
+  const {data, isLoading, error} = useApiQuery<GroupSummaryData>(
```

Comment:
mostly a nit but there's a isError instead of the error string
```suggestion
  const {data, isLoading, isError} = useApiQuery<GroupSummaryData>(
```